docs/suspensive.org/src/components/BubbleChart.tsx

```diff
@@ -1,6 +1,115 @@
+                  site_admin: boolean
+                }
+                total: number
+                weeks: {
+                  w: number // Week timestamp in epoch seconds
+                  a: number // Number of additions
+                  d: number // Number of deletions
+                  c: number // Number of commits
+                }[]
+              }[]
+            | undefined
+        }
+        throw new Error('Failed to fetch contributors')
+      }),
+  })
+
+export const BubbleChart = () => (
+  <QueryErrorBoundary
+    fallback={
+      <Link href="https://github.com/toss/suspensive/graphs/contributors">
+        <br />
+        <img
+          src="https://contrib.rocks/image?repo=toss/suspensive"
+          alt="contributors"
+          data-canonical-src="https://contrib.rocks/image?repo=toss/suspensive"
+          style={{ maxWidth: '100%' }}
+        />
+      </Link>
+    }
+  >
+    <Suspense clientOnly fallback={<></>}>
+      <SuspenseQuery {...contributorsQueryOptions()}>
+        {({ data }) => {
+          const chartData = data
+            ?.filter(
+              ({ author }) => !['github-actions[bot]', 'dependabot[bot]', 'renovate[bot]'].includes(author.login)
+            )
+            .map(({ author, total }) => ({
+              name: author.login,
+              value: total,
+              avatar: author.avatar_url,
+              htmlUrl: author.html_url,
+            }))
+
+          if (chartData) {
+            return (
+              <>
+                <div className="flex w-[100%] items-center justify-center overflow-visible sm:hidden md:hidden lg:hidden">
+                  <BubbleChartSize chartData={chartData} height={400} width={400} padding={2} />
+                </div>
+                <div className="hidden w-[100%] items-center justify-center overflow-visible sm:flex md:hidden lg:hidden">
+                  <BubbleChartSize chartData={chartData} height={630} width={630} padding={6} />
+                </div>
+                <div className="hidden w-[100%] items-center justify-center overflow-visible sm:hidden md:flex lg:hidden">
+                  <BubbleChartSize chartData={chartData} height={560} width={560} padding={4} />
+                </div>
+                <div className="hidden w-[100%] items-center justify-center overflow-visible sm:hidden md:hidden lg:flex">
+                  <BubbleChartSize chartData={chartData} height={760} width={760} padding={8} />
+                </div>
+              </>
+            )
+          }
+
+          return (
+            <Link href="https://github.com/toss/suspensive/graphs/contributors">
+              <br />
+              <img
+                src="https://contrib.rocks/image?repo=toss/suspensive"
+                alt="contributors"
+                data-canonical-src="https://contrib.rocks/image?repo=toss/suspensive"
+                style={{ maxWidth: '100%' }}
+              />
+            </Link>
+          )
```

Comment:
fallback for res.ok but [202](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202)!